### PR TITLE
add pcmsolver to osx_arm64 migration

### PR DESCRIPTION
**Pull request by loriab**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* ~Bumped the build number (if the version is unchanged)~
* ~Reset the build number to `0` (if the version changed)~
* ~[Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)~
* ~Ensured the license file is being packaged.~

### CODE REVIEW

File: recipe/migrations/osx_arm64.txt [hunk 1]
```diff
@@ -1275,3 +1275,4 @@ puma
 grpcio-tools
 kwn-deform
 helix
+pcmsolver-split
```
Comment (hmaarrfk) [inline recipe/migrations/osx_arm64.txt:1278]:
```suggestion
pcmsolver
```

Comment (hmaarrfk) [reply recipe/migrations/osx_arm64.txt:1278]: I think you want to specify the name of the packag,e and not the name of the recipe top level.

Comment (loriab) [reply recipe/migrations/osx_arm64.txt:1278]: I wasn't sure. `pcmsolver-split` is the feedstock name (yes, I made a mistake in not specifying `extra/feed-stockname` when the recipe got converted to multiout). The feedstock produces packages `pcmsolver` and `libpcm`.

Comment (hmaarrfk) [reply recipe/migrations/osx_arm64.txt]: Either one will get the migrating started i think

**Pull request merged**